projects/admin/src/app/service/ui-remote-typeahead.service.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2020 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { Injectable, Injector } from '@angular/core';
+import { RemoteTypeaheadService, RecordService, ApiService, SuggestionMetadata } from '@rero/ng-core';
+import { Observable } from 'rxjs';
+import { MefTypeahead } from '../class/mef-typeahead';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UiRemoteTypeaheadService extends RemoteTypeaheadService {
+
+  /** Custom typeahead Type */
+  private _typeaheadTypes = {
+    mef: this._injector.get(MefTypeahead)
+  };
+
+  /**
+   * Constructor
+   * @param _recordService - RecordService
+   * @param _apiService - APIService
+   * @param _injector - Injector
```

Comment:
can you put some comment about the use of this service?

Reply:
done